### PR TITLE
[12.0][FIX] web_widget_one2many_product_picker

### PR DESCRIPTION
**Pull request by Tardo**
- Apply fix made for 13.0 to set product_id via context to prevent do onchanges
- Fix parent record updates when uses 'auto_save' mode
- Fix "Lines" section to redraw lines when update qty
- Update ES translations

cc @tecnativa TT29553

### CODE REVIEW

**Comment by pedrobaeza** [comment]
TT?

File: web_widget_one2many_product_picker/i18n/es.po [hunk 1]
```diff
@@ -58,6 +64,11 @@ msgstr "Carga más"
 msgid "Price:"
 msgstr "Precio:"
 
+#. module: web_widget_one2many_product_picker
+#: model:ir.model,name:web_widget_one2many_product_picker.model_product_pricelist_item
+msgid "Pricelist Item"
+msgstr "Tarifa"
```
Comment (pedrobaeza) [inline web_widget_one2many_product_picker/i18n/es.po]: Elemento de tarifa

**Review by pedrobaeza — APPROVED** [review]

**Review by chienandalu — APPROVED** [review]
Tested :+1:

**Comment by Tardo** [comment]
@pedrobaeza changes done

**Pull request merged**